thaw/src/textarea/mod.rs

```diff
@@ -10,7 +10,8 @@ pub fn Textarea(
     #[prop(optional, into)] on_blur: Option<Callback<ev::FocusEvent>>,
     #[prop(optional, into)] disabled: MaybeSignal<bool>,
     /// Which direction the Textarea is allowed to be resized.
-    #[prop(optional, into)] resize: MaybeSignal<TextareaResize>,
+    #[prop(optional, into)]
+    resize: MaybeSignal<TextareaResize>,
```

Comment:
Some formatting changes. Is that ok or we have some different formatter settings?